node/Dockerfile.amd64

```diff
@@ -46,7 +46,9 @@ RUN dnf install -y 'dnf-command(config-manager)' && \
     # Install required packages for building rpms. yum-utils is not required but it gives us yum-builddep to easily install build deps.
     yum install --allowerasing -y rpm-build yum-utils make && \
     # Need these to build runit.
-    yum install --allowerasing  -y wget glibc-static gcc && \
+    yum install --allowerasing -y wget glibc-static gcc && \
```

Comment:
gcc is a heavy dep to install only to delete it again, can we not build the binary in go-build?

Reply:
This is a white space change. `gcc` is an existing dependency to build runit. I think the fips OpenSSL is a redhat additional to the vanilla OpenSSL project. We have to build the test problem in Redhat (and derived) OSs (in this case centos stage). Our go-build is Debian based and it is missing openssl dev pages and openssl/fips.h header.

Reply:
Sorry, my bad!